capsules/core/src/gpio.rs

```diff
@@ -198,8 +199,8 @@ impl<'a, IP: gpio::InterruptPin<'a>> SyscallDriver for GPIO<'a, IP> {
         let pins = self.pins.as_ref();
         let pin_index = data1;
         match command_num {
-            // number of pins
-            0 => CommandReturn::success_u32(pins.len() as u32),
+            // Check existence.
+            0 => CommandReturn::success(),
```

Comment:
I think we need PRs open for libtock-c and libtock-rs to make this change.

https://github.com/tock/libtock-c/blob/master/libtock/gpio.c
https://github.com/tock/libtock-rs/blob/master/apis/gpio/src/lib.rs#L64